objects/ArkhamSCE380-592024-Page1.bd6b3e.json

```diff
@@ -85,10 +85,10 @@
         "g": 1,
         "r": 1
       },
-      "Description": "- Implemented many miscellaneous bugfixes and metadata tweaks.\n\n- Extra-special thanks to everyone who helped us release Feast of Hemlock Vale as quickly as we could!!",
+      "Description": "Nkosi Mabati helper updated with custom token compatability.\nNew right-click options havee been added to Rite of Sanctification and Nepthys.\nMyriad other under-the-hood code optimizations and bugfixes have been included.\nFixed Ancient Covenant and Blasphemous Covenant having low-quality images.\n",
```

Comment:
Rite of Sanctification now just stays open during sealing ;)

Reply:
Tweaked the update notes!